development/src/composables/sessionManagementFunction.js

```diff
@@ -0,0 +1,124 @@
+import { generatePeerIdCharacter } from "./peerIdGenerator";
+import { toast } from "react-toastify";
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/database";
+import { firebaseConfig } from "./firebaseConfig/config";
```

Comment:
`firebaseConfig` does not seem to be used in this file? why the import?

Reply:
Will be used to check if users I authenticated and get user session data from firebase DB